src/ext/yuni/src/yuni/core/process/rename.h

```diff
@@ -1,28 +0,0 @@
-/*
```

Comment:
For all intent and purpose Yuni is not external since it is directly in the source code. Removing dead code allow:

- Smaller code base
- No risk to use it since the code is not maintained and we want to remove it
- Removed from static analyser
- Removed from global search (ctrl+f)